contrib/win32/openssh/ssh.vcxproj

```diff
@@ -507,6 +507,7 @@
     <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\win32_sshtty.c" />
     <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\win32-utf8.c" />
     <ClCompile Include="$(OpenSSH-Src-Path)ssh-sk-client.c" />
+    <ClCompile Include="$(OpenSSH-Src-Path)ssh-pkcs11.c" />
```

Comment:
Please revert this change. compile ssh-pkcs11.c as part of libssh project to align with upstream code.

Reply:
Removed. Thanks!